web/src/containers/activity/EntryDetails.js

```diff
@@ -1,8 +1,8 @@
 import { Button, Review } from '@cmsgov/design-system-core';
 import PropTypes from 'prop-types';
 import React, { Fragment, useMemo, useRef, useState } from 'react';
-import { connect } from 'react-redux';
 
+import { Provider } from './ActivityContext';
```

Comment:
Pull in an activity context provider. This is a custom thingy that does almost nothing on its own, but it wraps up some React functionality with a nice little bow to make it easier to use.